app/panel/components/BuildingBlocks/CliqzFeature.jsx

```diff
@@ -50,28 +50,9 @@ class CliqzFeature extends React.Component {
 		});
 	}
 
-	_getCount(active, data, type) {
```

Comment:
The data prop no longer needs to be passed down to CliqzFeture; we should remove it.

Reply:
....removed